app/routes/create-catch-certificate/$catchCertificate/what-are-you-exporting/whatAreYouExporting.ts

```diff
@@ -177,4 +216,21 @@ const onAddSpeciesResponse = async (response: Response, requestBody: any): Promi
     default:
       throw new Error(`Unexpected error: ${response.status}`);
   }
+}
+
+export const validateValues = (validate: string[], values: any): IError[] => {
+  const errors: IError[] = [];
+
+  if (validate?.length > 0) {
+    validate.forEach((key: string)  => {
+      if (!values[key]) {
+        errors.push({
```

Comment:
not a big fan of "push" because despite it's a new array you creating a mutation and I don't want that will be used as a pattern looking forward.

Reply:
Understood, thanks @daniele-zurico updated to remove mutation